build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+pushd frontend
+npm install
+npm install -g node-sass
+npm run-script build
+if [ ! -d ./dist/ ]; then
+    echo "Frontend build failed - no dist/ created"
+    exit 1
+fi
+echo "Frontend build successful"
+popd
+pushd server
+npm install
+pushd scripts
```

Comment:
Can we run `ln -s ../frontend/dist public` here? Some routing issues require us to access the dist directory without using a `../`